Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN go mod download && \
     GOOS=linux CGO_ENABLED=1 go build -ldflags="-linkmode external -extldflags='-static'" -o ipip_check -buildvcs=false && \
     gzip ipip_check
 
-FROM python:3.12-slim-bullseye AS python-build
+FROM python:3.12.4-slim-bullseye AS python-build
```

Comment:
H! Instead of downgrading to Python 3.12.4, I would recommend adding `setuptools` to the pip install command here (ie installing both `build` and `setuptools`):
https://github.com/Netcracker/KubeMarine/blob/1b3e612e1eff7fbdcace47bbfc58ac8636ad8c0b/Dockerfile#L28

And then longer term I would report a bug with whatever dependency is using setuptools but hasn't correctly listed it in its requirements (or similar).

Reply:
Hi, @edmorley!
Initially, I wanted to fix the patch version to avoid such issues in the future, because they can break pipelines during the development of other features (like now). Updating the version is our regular task, so we would not get stuck on `3.12.4`.
 But after a private discussion with @koryaga , it was decided to leave everything as is, and resolve only this particular issue.